package.json

```diff
@@ -95,11 +95,10 @@
     "prettier-plugin-organize-imports": "4.1.0",
     "source-map-support": "0.5.21",
     "ts-node": "10.9.2",
-    "typedoc": "0.25.13",
-    "typedoc-plugin-carbon-ads": "1.6.0",
```

Comment:
We might need to make a fork of this, since we do get a bit of money from it

Reply:
Yup! However from my quick check it seemed to already be broken on the current version. It did not appear on the typedoc pages

Reply:
![Screenshot_2025-03-08-21-50-30-92_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/user-attachments/assets/2d5b681a-5d8a-40b5-9323-7e8a27107d00)

Looks fine on mobile for me

Reply:
Must be a config in my browser, even without ublock it's not there for me
But good to know!

Reply:
We can update TS first, I just looked at the code and we might also just copy the file over instead of forking the entire repo